Embedded Metadata.js

```diff
@@ -596,6 +597,11 @@ function processHighwireCreators(creatorNodes, role, doc) {
 			// skip empty authors. Try to match something other than punctuation
 			if (!creator || !creator.match(/[^\s,-.;]/)) continue;
 
+			// Skip adjacent repeated authors
+			if (lastCreator && creator == lastCreator) continue;
```

Comment:
the only scenario in which I could see authors being repeated are different roles (i.e. editor & author being the same person, say in the intro to an edited volume). But `role` is inherited from the function call, so that doesn't seem relevant here.